config.md

```diff
@@ -317,6 +320,10 @@ Annotations are key-value maps.
 }
 ```
 
+## Extensibility
+The `annotations` property MAY be used as an extensibility point to include additional information that is not defined as part of this specification.
+Excluding the `annotations` property, implementations that are reading/processing this configuration file MUST generate an error if they encounter an unkown property.
```

Comment:
This seems unnecessarily strict.  The `ociVersion` field uniquely (ish, #371) declares semantics for fields defined by this spec.  Maybe the additional properties are optional features (folks have talked about [having both Solaris and Linux sections in the same config][1], although that particular use-case doesn't make much sense to me).  In any case, I think this should be relaxed to SHOULD, and would recommend runtimes provide an `--unknown-properties=(ignore|warn|error)` flag or some such to allow callers to adjust the behavior.

More generally, I think JSON-LD is a [good][2] [way][3] to assign semantics to all properties in the config, whether they are stock OCI properties or extensions.  That would give callers a better chance of figuring out whether a given extension was ignorable or not by making it easier to track down its semantics.

[1]: https://github.com/opencontainers/runtime-spec/pull/411#discussion_r61623026
[2]: https://github.com/opencontainers/runtime-spec/pull/371#issuecomment-209684002
[3]: https://github.com/opencontainers/image-spec/pull/111#discussion_r65619280